vendor/knative.dev/eventing/pkg/channel/event_receiver.go

```diff
@@ -63,7 +62,7 @@ type ReceiverOptions func(*EventReceiver) error
 
 // ResolveChannelFromHostFunc function enables EventReceiver to get the Channel Reference from incoming request HostHeader
 // before calling receiverFunc.
-type ResolveChannelFromHostFunc func(url.URL) (ChannelReference, error)
+type ResolveChannelFromHostFunc func(string) (ChannelReference, error)
```

Comment:
I think this change will need to be propagated to consuming code in this PR too

Reply:
Yeah looks like it =/.  I was trying to keep them separate initially. Updated the PR!